README.md

```diff
@@ -26,7 +26,6 @@ Built with React, Node, PouchDB/CouchDB and using offline first design, we're wo
 - [Contributing](#contributing)
 - [Project Structure](#project-structure)
 - [Application Infrastructure](#application-infrastructure)
-- [Monorepo-specific Contributing Guide](#monorepo-specific-contributing-guide)
```

Comment:
I'm not so sure I want to remove this chapter

Reply:
Do you mean, it might be better staying in README than moved to CONTRIBUTING?

Reply:
Maybe not, never mind.